target/linux/ar71xx/base-files/etc/board.d/02_network

```diff
@@ -424,7 +424,8 @@ ar71xx_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan:1" "2:lan:2" "3:lan:3" "4:lan:4"
 		;;
-	tl-wr842n-v2)
+	tl-wr842n-v2|\
+	tl-wr942n-v1)
 		ucidef_set_interfaces_lan_wan "eth1.1" "eth0"
 		ucidef_add_switch "switch0" \
 			"0@eth1" "1:lan:4" "2:lan:1" "3:lan:2" "4:lan:3"
```

Comment:
> Question here is what is more important for the user: working USB LEDs or serial console and what we should support by default if there is no way to have both things working at the same time.

Can't this be set that if no serial is detected on boot it will switch to controlling USB leds?